physical/raft/raft.go

```diff
@@ -902,6 +918,58 @@ func (b *RaftBackend) SetupCluster(ctx context.Context, opts SetupOpts) error {
 	// Close the init channel to signal setup has been completed
 	close(b.raftInitCh)
 
+	reloadConfig := func() {
+		newCfg := raft.ReloadableConfig{
+			TrailingLogs:      raftConfig.TrailingLogs,
+			SnapshotInterval:  raftConfig.SnapshotInterval,
+			SnapshotThreshold: raftConfig.SnapshotThreshold,
+			HeartbeatTimeout:  raftConfig.HeartbeatTimeout / initialTimeoutMultiplier,
+			ElectionTimeout:   raftConfig.ElectionTimeout / initialTimeoutMultiplier,
+		}
+		err := raftObj.ReloadConfig(newCfg)
+		if err != nil {
+			b.logger.Error("failed to reload raft config to set lower timeouts", "error", err)
+		} else {
+			b.logger.Trace("reloaded raft config to set lower timeouts", "config", fmt.Sprintf("%#v", newCfg))
+		}
+	}
+	confFuture := raftObj.GetConfiguration()
+	if err := confFuture.Error(); err != nil {
+		b.logger.Error("failed to read raft configuration", "error", err)
+	}
+	clusterConf := confFuture.Configuration()
```

Comment:
If the error condition is triggered above, on line 937, and we log an error on line 938, what happens here? I don't know enough about the raft library and its use of futures to reason about this effectively, but if `confFuture.Error()` is not nil, will `confFuture.Configuration()` hold a meaningful value?

Reply:
D'oh, good point.

Reply:
Fixed.